README.md

```diff
@@ -12,29 +12,29 @@
 
 # Workday HCM dbt Package ([Docs](https://fivetran.github.io/dbt_workday/))
 
-# 📣 What does this dbt package do?
+## What does this dbt package do?
 
 This package models Workday HCM data from [Fivetran's connector](https://fivetran.com/docs/applications/workday-hcm). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/workday-hcm#schemainformation).
 
 The main focus of the package is to transform the core object tables into analytics-ready models, including:
 <!--section="workday_model"-->
```

Comment:
@fivetran-joemarkiewicz should this section exist, or just the `workday_transformation_model` section on line 33?

Reply:
or should there be another `<!--section-end-->`?